sqlagg/base.py

```diff
@@ -143,7 +143,7 @@ def _build_query(self, metadata):
                     elif group_key in alias:
                         aliased_column = [col.build_column(table) for col in self.columns if col.alias == group_key]
                         assert len(aliased_column) == 1
```

Comment:
since this is only testing the length of the list we don't need to call `col.buid_column(table)` in the line above.

Reply:
Actually I think I can just remove both of these lines, yeah?  `aliased_column` isn't used anywhere else, and it doesn't look like `build_column` mutates anything.

Reply:
Or do you think it's useful to have a check there that there's precisely one `alias` that matches that `group_key`

Reply:
I think it could be useful to have that check